lib/solr_wrapper/configuration.rb

```diff
@@ -7,10 +7,13 @@ class Configuration
     def initialize(options = {})
       @config = options[:config]
       @verbose = options[:verbose]
-
       @options = load_configs(Array(options[:config])).merge(options)
     end
 
+    def debug?
+      @debug
```

Comment:
```suggestion
      options[:debug]
```

maybe? I'm not seeing where `@debug` is getting set.